test/test_randomforest.jl

```diff
@@ -85,7 +85,7 @@ end
     test_labels = Vector{Float64}(y[test_indices])
     train_data = Matrix(X[train_indices, :])
     test_data = Matrix(X[test_indices, :])
-    forest = RandomForest(split_ig, 10, 0.9, 10)
+    forest = RandomForest(split_variance, 10, 0.9, 10)
```

Comment:
This should have been split_variance from the start since it is a regression task